build/ubuntu/Dockerfile.intermediate

```diff
@@ -0,0 +1,18 @@
+ARG builder=quay.io/wire/ubuntu20-builder
```

Comment:
why did you remove the docs comments here?  (just curious)

Reply:
Added them back.